python/setup.py

```diff
@@ -102,6 +102,16 @@ def parallelCCompile(
     macros, objects, extra_postargs, pp_opts, build = self._setup_compile(
         output_dir, macros, include_dirs, sources, depends, extra_postargs)
     cc_args = self._get_cc_args(pp_opts, debug, extra_preargs)
+
+    if NGRAPH_PYTHON_DEBUG in ['TRUE', 'ON', True]:
+        try:
+            # pybind11 is much more verbose without -DNDEBUG
```

Comment:
That's weird.

Reply:
But unfortunately that's true. When I made sure that this flag is present, then I much faster managed to find out what the problem was. That's just debugging convenience.